GitDiffMargin/GitDiffMargin.csproj

```diff
@@ -66,9 +67,7 @@
       GitDiffMargin.Commands references this project using a ProjectReference.
     -->
     <PackageReference Include="LibGit2Sharp" Version="0.26.2" PrivateAssets="all" />
-    <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="2.0.306" ExcludeAssets="build" PrivateAssets="all">
-      <IncludeAssets>runtime; compile; native; contentfiles; analyzers</IncludeAssets>
-    </PackageReference>
+    <PackageReference Include="LibGit2Sharp.NativeBinaries" GeneratePathProperty="true" Version="2.0.306" ExcludeAssets="build" PrivateAssets="all" />
```

Comment:
📝 Updating NuGet to 5.3.1 means `GeneratePathProperty="true"` is recognized and produces a new build property `PkgLibGit2Sharp_NativeBinaries` which I used for referencing the package location in the cache.

Reply:
Nice [learning](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#generatepathproperty)